scss/modules/_nav.scss

```diff
@@ -204,7 +204,7 @@
     .site-nav__link {
 
       &.is-parent .site-nav__link__title {
-        border-bottom-color: $secondary;
+        border-bottom-color: $deep-red;
```

Comment:
@noahmanger thank you for hopping on this so fast. One question, I noticed that below on [line 241 ](https://github.com/18F/fec-style/pull/719/files#diff-a7b71e5fa78e81ccd516cf3142ae146eR241) for the corresponding sections in blue, it uses `border-color` instead of `border-bottom-color`. Should they match?